graphql_compiler/compiler/expressions.py

```diff
@@ -120,6 +120,7 @@ def _to_output_code(self):
 
     to_gremlin = _to_output_code
     to_match = _to_output_code
+    to_cypher = _to_output_code
```

Comment:
I couldn't select _to_output_code() line 98, this is the closest I could select.
Update the documentation of _to_output_code to reflect our support of Cypher.